code/game/machinery/doors/airlock.dm

```diff
@@ -350,6 +349,10 @@
 		if (cyclelinkedairlock.cyclelinkedairlock == src)
 			cyclelinkedairlock.cyclelinkedairlock = null
 		cyclelinkedairlock = null
+	if(close_others) //remove this airlock from the list of every linked airlock
+		closeOtherId = null
+		for(var/obj/machinery/door/airlock/otherlock as anything in close_others)
+			otherlock.close_others -= src
 	if(id_tag)
```

Comment:
.Cut() the list

Reply:
Reread these lines @Cyberboss

Reply:
wait, @Cyberboss are you suggesting they change the for to a `.Cut()` or add a Cut below it.

Reply:
no idea what that means, you would have to explain. im just doing the same thing the current helper does

Reply:
close_others.Cut after for loop

Reply:
> 
> 
> close_others.Cut after for loop

otherlock.close_others.Cut(src)  ?

Reply:
No, src.close_others, which has references to other airlocks

Reply:
```suggestion
	if(close_others) //remove this airlock from the list of every linked airlock
		closeOtherId = null
		for(var/obj/machinery/door/airlock/otherlock as anything in close_others)
			otherlock.close_others -= src
		close_others.Cut()
	if(id_tag)
```

Reply:
hmmm, im not sure exactly what this even does but ill add it in and test to ensure it doesnt cause any issues, doubt it will but testing is always good